util/logutil/log.go

```diff
@@ -44,6 +44,8 @@ const (
 	DefaultRecordPlanInSlowLog = 1
 	// DefaultTiDBEnableSlowLog enables TiDB to log slow queries.
 	DefaultTiDBEnableSlowLog = true
+	// DefaultExpensiveScanRatio is the ratio of iterated keys and processed keys.
+	DefaultExpensiveScanRatio = 20
```

Comment:
If the ratio equals how many physical keys this query has touched, then 100K is a more reasonable default expensive threshold.

Reply:
The ratio is closer to the nature of the MVCC version problem; however we may want to add the count threshold to avoid the log flooding.  And I think ratio 20x can be lifted to 100x.

Reply:
The default value is increased to 100.